client/java-armeria-legacy/src/main/java/com/linecorp/centraldogma/client/armeria/legacy/LegacyCentralDogmaBuilder.java

```diff
@@ -60,7 +63,15 @@ public CentralDogma build() throws UnknownHostException {
             }
             return delegate.execute(ctx, req);
         });
-        return new LegacyCentralDogma(clientFactory().eventLoopGroup(),
-                                      builder.build(CentralDogmaService.AsyncIface.class));
+
+        final EventLoopGroup executor = clientFactory().eventLoopGroup();
+        final int maxRetriesOnReplicationLag = maxNumRetriesOnReplicationLag();
+        final CentralDogma dogma = new LegacyCentralDogma(executor, builder.build(AsyncIface.class));
+        if (maxRetriesOnReplicationLag <= 0) {
```

Comment:
is it possible that maxRetriesOnReplicationLag < 0 ?
maxNumRetriesOnReplicationLag(int maxRetriesOnReplicationLag) already checked the augment.

Reply:
It's not possible but I just wanted to be as defensive as possible. :-)

Reply:
oh, I see